src/NeuralNetDiffEq.jl

```diff
@@ -58,13 +58,95 @@ function Base.show(io::IO, A::KolmogorovPDEProblem)
   show(io , A.sigma)
 end
 
+struct GeneranNN2DimPDEProblem{PF,BC,IC,T,X,DT,DX,P} <:DiffEqBase.DEProblem
```

Comment:
why does 2D and 3D need to be treated differently?

Reply:
it is old code. I going to delete it soon.